packages/components/pie-button/package.json

```diff
@@ -10,6 +10,7 @@
     "build": "run -T vite build",
     "watch": "run -T vite build --watch",
     "test": "echo \"Error: no test specified\" && exit 0",
+    "test:ci": "yarn test",
```

Comment:
This should not be needed. CI systems have an environment variable called `CI` set to `true`. Quality tools work with that variable natively. This includes Vitest. 

Usually in a modern repo I'd expect two jobs:
```
    "test": "vitest",
    "coverage": "vitest --coverage",
```

CI would then simply `run: yarn coverage`

Reply:
this is fair point @webbertakken but @siggerzz please raise a separate PR if you wish to address this :)